events/wc_frostwolf_events.txt

```diff
@@ -1,55 +1,86 @@
 namespace = WCFW
-#Frost Wolf exile event chain 1-99
-# Delayer event
+# Frostwolf Chain
+# Delayer event = delayed event + 100
+# Reaction event = actual event + 200
+
+# Starting Chain 1-3
+# delayer event for starting chain
 province_event = {
 	id = WCFW.101
 
 	is_triggered_only = yes
 	hide_window = yes
 
 	immediate = {
-		d_frostwolf = {
+		d_stormreaver = {
 			holder_scope = {
-				if = {
-					limit = { frostwolf_exodus_d_frostwolf_trigger = yes }
+				if = { # Only start this if there's a butcher alive, if not then.. nothing happens!
+					limit = { frostwolf_chain_butcher_exists = yes }
 					narrative_event = { id = WCFW.1 }
 				}
-				else = {
-					character_event = { id = WCFW.200 }
-				}
 			}
 		}
 	}
 }
-# Delayed event
+
+# Event to Stormreaver owner (Gul'dan or successors)
 narrative_event = {
 	id = WCFW.1
 	title = EVTTITLE_WCFW_1
 	desc = EVTDESC_WCFW_1
 	picture = GFX_evt_orc_city
 	border = GFX_event_narrative_frame_war
 
+	is_triggered_only = yes
 	hide_from = yes
 
+	option = {
+		name = EVTOPT_WCFW_1
+		custom_tooltip = { text = EVTOPT_WCFW_1_tooltip }
+		73 = {
+			province_event = {
+				id = WCFW.200
+				days = 7
+			}
+		}
+	}
+}
+
+# Delayer event
+province_event = {
+	id = WCFW.200
```

Comment:
Shouldn't ID be `WCFW.102`?

Reply:
Very much so, it seems I missed the first few ID changes. I will resolve this.

Reply:
Resolved in latest commit.